### DISCUSSION TITLE
寻求帮助！在安装完nonebot之后运行nb run出现文件不存在

### DISCUSSION THREAD

**Discussion by HX-Wrdzgzs** [nonebot/discussions#535] [问答]
![Screenshot_20241125_195107_com_microsoft_rdc_androidx_SessionActivity_edit_295133546678768](https://github.com/user-attachments/assets/84291203-58e9-46c3-b1f2-6df05f913162)
![Screenshot_20241125_195751_com_microsoft_rdc_androidx_SessionActivity_edit_295147764881887](https://github.com/user-attachments/assets/55134aab-9d80-47b0-8189-5851175fa7ed)
![Screenshot_20241125_195749_com_microsoft_rdc_androidx_SessionActivity_edit_295155720664541](https://github.com/user-attachments/assets/15cbc00c-ab78-4709-acf2-fa02888ef0f5)
这是第三次出现文件不存在，我以为是pipx问题，将pipx重新安装过后依旧存在，想问下大佬这种问题该如何解决

**Comment by yanyongyu** [comment]
你应该在你的bot项目目录下运行nb run，也就是pyproject.toml文件所在目录

**Reply by HX-Wrdzgzs** [reply]
谢谢您的回答，问题已解决
